addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1257,6 +1257,30 @@ const InputUserValueWidget = UnitUserValueWidget.extend({
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+     * @override
+     */
+    getActiveValue() {
+        const params = this._methodsParams;
+        if (!params.max && !params.min) {
+            return this._super(...arguments);
+        }
```

Comment:
Ah yes indeed, I totally forgot about 0 :grimacing: Because when using the inputs, it often puts 0 when writing nothing or negative number so I thought that specifying `data-min="0"` would never be done haha 
But it is not the case for all the inputs so I should have considered it, thanks :+1: